replace.json

```diff
@@ -0,0 +1,3 @@
+{
+  "/* @echo FONT_CDN_PATH *":"https://dhrlmnmyf2njb.cloudfront.net"
```

Comment:
Tried to get this working using an environment variable. @teneightfive, maybe we could pair on this?